src/js/mep-feature-progress.js

```diff
@@ -246,9 +246,10 @@
 				percent = null;
 
 			// newest HTML5 spec has buffered array (FF4, Webkit)
-			if (target && target.buffered && target.buffered.length > 0 && target.buffered.end && target.duration) {
-				// account for a real array with multiple values - always read the end of the last buffer
-				percent = target.buffered.end(target.buffered.length - 1) / target.duration;
+			const buffer = target && target.buffered; 
+			if (buffer && target.buffered.length > 0 && target.buffered.end && target.duration) {
```

Comment:
still referencing `target.buffered` a couple of times here.  should just deal with the `buffer` reference instead